Assets/Huawei/Editor/Utils/HMSGradleFixer.cs

```diff
@@ -38,6 +57,9 @@ public void OnPostGenerateGradleAndroidProject(string path)
 
         string baseProjectTemplatePath = Application.dataPath + "/Huawei/Plugins/Android/hmsBaseProjectTemplate.gradle";
         FileUtil.CopyFileOrDirectory(baseProjectTemplatePath, Directory.GetParent(path).FullName + @"/hmsBaseProjectTemplate.gradle");
+
+        GradleVersionFixer(File.ReadAllText(Directory.GetParent(path).FullName + "/build.gradle"), path);
```

Comment:
Maybe we can if condition for Account kit or Push kit is enable Which need to update build Gradle version for it.